Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Python image to use.
```

Comment:
Using an alpine image can lead to unexpected issues if your application relies on specific libraries or system dependencies not available in the reduced alpine environment.  Consider using a slim variant if image size is a concern.